docs/usage.md

```diff
@@ -1,153 +1,306 @@
+- [Run nf-core/raredisease with your data](#run-nf-core-raredisease-with-your-data)
+  - [Samplesheet](#samplesheet)
+  - [Reference files and parameters](#reference-files-and-parameters)
+    - [1. Alignment](#1-alignment)
+    - [2. QC stats from the alignment files](#2-qc-stats-from-the-alignment-files)
+    - [3. Repeat expansions](#3-repeat-expansions)
+    - [4. Variant calling - SNV](#4-variant-calling---snv)
+    - [5. Variant calling - Structural variants](#5-variant-calling---structural-variants)
+    - [6. SNV annotation & Ranking](#6-snv-annotation---ranking)
+    - [7. SV annotation & Ranking](#7-sv-annotation---ranking)
+    - [8. Mitochondrial analysis](#8-mitochondrial-analysis)
+  - [Run the pipeline](#run-the-pipeline)
+    - [Direct input in cli](#direct-input-in-cli)
+    - [Import from a config file (recommended)](#import-from-a-config-file--recommended-)
+- [Best practices](#best-practices)
+- [Troubleshooting](#troubleshooting)
+  - [Resource errors](#resource-errors)
+    - [For beginners](#for-beginners)
+    - [Advanced option on process level](#advanced-option-on-process-level)
+- [Custom configuration](#custom-configuration)
+  - [Updating containers (advanced users)](#updating-containers--advanced-users-)
+  - [nf-core/configs](#nf-core-configs)
+  * [Run sentieon](#run-sentieon)
+  * [Azure Resource Requests](#azure-resource-requests)
+  * [Running in the background](#running-in-the-background)
+  * [Nextflow memory requirements](#nextflow-memory-requirements)
+
+# Introduction
+
+nf-core/raredisease is a bioinformatics best-practice analysis pipeline to call, annotate and score variants from WGS/WES of rare disease patients. The pipeline is built using nextflow.
+
+# Prerequisites
+
+1. Install Nextflow (>=22.10.1) using the instructions [here.](https://nextflow.io/docs/latest/getstarted.html#installation)
+2. Install one of the following technologies for full pipeline reproducibility: Docker, Singularity, Podman, Shifter or Charliecloud.
+   > Almost all nf-core pipelines give you the option to use conda as well. However, some tools used in the raredisease pipeline do not have a conda package so we do not support conda at the moment.
+
+# Run nf-core/raredisease with test data
+
+Before running the pipeline with your data, we recommend running it with the test dataset available [here](https://github.com/nf-core/test-datasets/tree/raredisease).
+
+> You do not need to download the data as the pipeline is configured to fetch that data automatically for you when you use the test profile.
+
+Run the following command, where YOURPROFILE is the package manager you installed on your machine. For example, `-profile test,docker` or `-profile test,singularity`:
 
-> _Documentation of pipeline parameters is generated automatically from the pipeline schema and can no longer be found in markdown files._
+```
+nextflow run nf-core/raredisease \
+    -revision dev -profile test,<YOURPROFILE> \
```

Comment:
When this goes to master the `-revision` parameter should reflect that. Can we make it clear somehow

Reply:
Yupe! Will update the document after we get community approval and before we make an actual release.